Food/jreleaser.lua

```diff
@@ -33,17 +36,57 @@ food = {
                 }
             }
         },
+        {
+            os = "darwin",
+            arch = "arm64",
+            url = "https://github.com/" .. name .. "/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-standalone-" .. version .. "-osx-aarch64.zip",
+            sha256 = "1881fc7a88fd0ea687c9ff96aaca17821744ab5fb0813a85462578f990419c37",
+            resources = {
+                {
+                    path = name .. "-standalone-" .. version .. "-osx-aarch64/bin/" .. name,
+                    installpath = "bin/" .. name,
+                    executable = true
+                }
+            }
+        },
+        {
+            os = "darwin",
+            arch = "amd64",
+            url = "https://github.com/" .. name .. "/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-standalone-" .. version .. "-osx-x86_64.zip",
+            sha256 = "184540ecf756d688babbed32ef520042290fad21b5a38266d15143cc5d101439",
+            resources = {
+                {
+                    path = name .. "-standalone-" .. version .. "-osx-x86_64/bin/" .. name,
+                    installpath = "bin/" .. name,
+                    executable = true
+                }
+            }
+        },
+        {
+            os = "windows",
+            arch = "arm64",
+            url = "https://github.com/" .. name .. "/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-standalone-" .. version .. "-windows-aarch64.zip",
+            sha256 = "bd7b4c780ee4f5a3c3ad8ea5f6eb9a74fc3d50ec07a59b39d40918bfebc31a11",
+            resources = {
+                {
+                    path = name .. "-standalone-" .. version .. "-windows-aarch64\\bin\\" .. name .. ".bat",
+                    installpath = "bin\\" .. name .. ".bat",
+                    executable = true
+                }
+            }
+        },
         {
             os = "windows",
             arch = "amd64",
             url = "https://github.com/" .. name .. "/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-standalone-" .. version .. "-windows-x86_64.zip",
-            sha256 = "5b9b7616b948aebeaef158cc62d4a5c512ecce5e8f1dd5d49758e1a341177f35",
+            sha256 = "3f3948542208d297d794342e109df3f0c1c023d4142024688125a2aa9b9c7a72",
             resources = {
                 {
                     path = name .. "-standalone-" .. version .. "-windows-x86_64\\bin\\" .. name .. ".bat",
-                    installpath = "bin\\" .. name .. ".bat"
+                    installpath = "bin\\" .. name .. ".bat",
+                    executable = true
```

Comment:
Not necessary for windows targets

Reply:
I noticed `executable = true` was not specified before but when I looked at other food entries I think I saw it explicitly set. I can remove it of course.

Reply:
Removed and squashed.